src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -390,21 +390,35 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
 #Region "Buffer helpers"
 
+        Private Function NextAre(chars As String) As Boolean
+            Debug.Assert(Not String.IsNullOrEmpty(chars))
+            Dim n = chars.Length-1
```

Comment:
@pharring Result 6 failures, if we use `AreNext(0, chars)` 
So this could suggest that the implementation of `AreNext` is incorrect.
If it is, it likely to do with this check.
`If Not CanGet(offset + n) Then Return False`
so I'll try
`If Not CanGet(offset + n-1) Then Return False`

and retest.